tools/generate_dynamic_content.py

```diff
@@ -0,0 +1,196 @@
+or if you have a suggestion, <a href="https://www.rudder-project.org/redmine/projects/rudder/issues/new?issue[category_id]=19">open an issue</a> in our bugtracker.</p>
+
+</td></tr></table></div>
+                   """)
+
+  output.append("<h2>Getting Started</h2>")
+
+  output.append("<p>If you are new to Rudder you can follow these steps to discover and start using Rudder:</p>")
+
+  output.append("<ul>")
+  output.append("<li>Read <a href=\"_introduction.html\">introduction</a> to get familiar with Rudder architecture and concepts.</li>")
+  output.append("""<li>Follow the <a href=\"_installation.html\">installation</a> section wich explains the <a href=\"configure_the_network.html\">requirements</a> and the installation steps for the different
+                 types of nodes:</li>""")
+  output.append("<ul>")
+  output.append("<li>The central <a href=\"_install_rudder_server.html\">server</a> </li>")
+  output.append("<li>The managed <a href=\"install-agent.html\">nodes</a></li>")
+  output.append("</ul>")
+  output.append("<li>Learn how to use the <a href =\"_rudder_web_interface.html\">Web interface</a>, to <a href=\"_node_management_2.html\">manage nodes</a> and to <a href=\"_configuration_management_2.html\">configure them</a>.</li>")
+  output.append("</ul>")
+
+  output.append("<h2>Resources</h2>")
+
+  output.append("""<p>Besides this documentation, you can find a lot of other useful information:</p>""")
+
+  output.append("<ul>")
+  output.append("<li>The <a href=\"http://www.rudder-project.org/changelog-" + version_info["version"] + "\">Changelog</a> for this version. It describes the changes in each release.</li>")
+  output.append("<li>The <a href=\"https://www.rudder-project.org/site\">Rudder project website</a>, particularly:")
+  output.append("<ul>")
+  output.append("<li>What is an ESR version? How long are Rudder versions supported? All the answers are in the <a href=\"https://www.rudder-project.org/site/documentation/faq/#versioning\">versionning policy</a>.")
+  output.append("</ul>")
+  output.append("<li>You need help or have questions about Rudder? You can:</li>")
+  output.append("<ul>")
+  output.append("<li>Send an email to the user <a href=\"https://www.rudder-project.org/site/support/forums-lists/\">mailing-list</a></li>")
+  output.append("<li>Join the <a href=\"https://www.rudder-project.org/site/support/chat-mailing-lists/\">IRC channel</a></li>")
+  output.append("</ul>")
+  output.append("<li>You need training,  or commercial support? Normation provides <a href=\"http://www.normation.com/en/rudder/services/\">services</a> for Rudder</li>")
+  output.append("<li>Browse the <a href=\"http://www.rudder-project.org/foswiki/Development/WebHome\">developper documentation</a> if you want to contribute or dive into Rudder internals</li>")
+  output.append("</ul>")
```

Comment:
Does this rather large block of content have to in a Python file? Couldn't we have it elsewhere and include it at least?